ui/src/app/applications/components/application-fullscreen-logs/application-fullscreen-logs.tsx

```diff
@@ -1,11 +1,15 @@
 import * as React from 'react';
+import Helmet from 'react-helmet';
 import {RouteComponentProps} from 'react-router-dom';
 import {PodLogsProps, PodsLogsViewer} from '../pod-logs-viewer/pod-logs-viewer';
 import './application-fullscreen-logs.scss';
 
 export const ApplicationFullscreenLogs = (props: RouteComponentProps<PodLogsProps>) => {
+    const title = `${props.match.params.podName}/${props.match.params.containerName}`;
     return (
         <div className='application-fullscreen-logs'>
+            <Helmet title={`Argo CD - ${title}`} />
```

Comment:
Are you sure this shouldn't be:
```suggestion
            <Helmet title={`${title} - Argo CD`} />
```

Note that users will have dozens of Argo CD tabs, so the distinct info really should be at the head of the title.

Reply:
Good point @jsoref . I agree